### PR TITLE
Project Review

### PR DESCRIPTION
**Pull request by HeflerDev**
Hello reviewer!
Deploying the first attempt of this project, happy review!

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+class UsersController < ApplicationController
+  def new
+  end
```
Comment (Torres-ssf) [inline app/controllers/users_controller.rb]: In order to use the `form_for` you will need to instantiate a blank User object and store it an instance called `@user`
Screenshot from 2020-03-02 21-41-56

File: app/views/users/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+<form action="/users" method="post" accept-charset='UTF-8'>
+<input type='hidden' name='authenticity_token' value='<%= form_authenticity_token%>'>
+  <label for="uname">Username:</label>
+  <input type="text" id="uname" name="user[username]"><br><br>
+  <label for="email">E-mail:</label>
+  <input type="text" id="email" name="user[email]"><br><br>
+  <label for="pass">Password</label>
+  <input type='password' id='pass' name='user[password]'><br><br>
+  <input type="submit" value="Submit">
+</form>
```
Comment (Torres-ssf) [inline app/views/users/new.html.erb]: You built the first form, nice :+1: 
But this one, along with the second needs to be commented out.
Screenshot from 2020-03-02 21-37-29
Remember, you need to use the last form which is the last implemented in the odin lesson.
Screenshot from 2020-03-02 21-41-56

File: config/routes.rb [hunk 3]
```diff
@@ -1,3 +1,4 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  resources :users, only: [:new, :create]
```
Comment (Torres-ssf) [inline config/routes.rb]: We need to add routes also for `edit` and `update`.
Screenshot from 2020-03-02 21-34-12

**Comment by HeflerDev** [comment]
Thanks, @Torres-ssf, for your review. I read it your feedback **but will begin the project again with my partner**, I rushed in front and didn't see that this project wasn't a mockup like the previous ones and was collaborative. I'm sure I will take care of all you pointed out in this review on my next submission. Wish you the best!

**Pull request merged**